src/data/finset/basic.lean

```diff
@@ -53,9 +53,6 @@ and the empty finset otherwise. See `data.fintype.basic`.
   then it holds for the finset obtained by inserting a new element.
 * `finset.choose`: Given a proof `h` of existence and uniqueness of a certain element
   satisfying a predicate, `choose s h` returns the element of `s` satisfying that predicate.
-* `finset.card`: `card s : ℕ` returns the cardinalilty of `s : finset α`.
-  The API for `card`'s interaction with operations on finsets is extensive.
-  TODO: The noncomputable sister `fincard` is about to be added into mathlib.
```

Comment:
It might be worth mentioning `finset.card` elsewhere in this module docstring to provide an easy link to its definition in the other file.

Reply:
Agreed!

Reply:
Done!